k8s/webhooks.go

```diff
@@ -197,7 +197,7 @@ func (w *WebhookServer) HandleValidateHTTP(writer http.ResponseWriter, req *http
 	}
 
 	// Run the controller
-	err = controller.Validate(admReq)
+	err = controller.Validate(context.Background(), admReq)
```

Comment:
We should be passing the real context here:
```suggestion
	err = controller.Validate(req.Context(), admReq)
```